### ISSUE TITLE
aarch64 runs for torchbench benchmarks

### ISSUE THREAD

**Issue by snadampal** [pytorch/benchmark#1463]
Hi, I would like to check if there are any nightly runs for these benchmarks.  If yes, I would like to enable few environment variables for aarch64 platforms. Please let me know where to contribute. Thank you!

**Comment by xuzhao9** [comment]
Yes, we have nightly runs on an Amazon g4dn.metal instance: https://github.com/pytorch/benchmark/actions/workflows/v2-nightly.yml. Currently, we don't have any aarch64 GitHub runner in our infra. I am not sure if you are planning to add new runners on the AWS side, or do you want to reuse the runners in the pytorch/pytorch infra to run the nightly workflow?

**Comment by snadampal** [comment]
thanks for the prompt response, @xuzhao9 . I wanted to understand if they are already running. Probably good option would be just use pytorch/pytorch infra as an extended CI.
